src/app/pages/home/footer/footer.component.html

```diff
@@ -1 +1,31 @@
-<div>Footer Page</div>
\ No newline at end of file
+
+<nav class=" navbar navbar-dark bg-dark mt-10 clearfix">

```

Comment:
Suggest you wrap everything inside the `.navbar` component inside a `<div class="container">...</div>` first. This should allow Bootstrap's responsive sizing to kick in. With that, you should be able to remove the `clearfix` class along with the `m-auto navbar-expand` classes from the respective children.

Should also be safe to just use a `div` tag instead of `nav` and also to remove the `mt-10` class which doesn't achieve anything.
```suggestion
<div class="navbar navbar-dark bg-dark">
  <div class="container">
    ...
  </div>
</div>
```